buildpipeline/pipeline.json

```diff
@@ -45,12 +45,13 @@
           }
         },
         {
-          "Name": "Core-Setup-Linux-BT",
+          "Name": "Core-Setup-Linux-Arm-BT",
           "Parameters": {
             "PB_DistroRid": "alpine.3.6-x64",
             "PB_DockerTag": "alpine-3.6-3148f11-20171119021156",
             "PB_TargetArchitecture": "x64",
-            "PB_AdditionalBuildArguments":"-TargetArchitecture=x64 -DistroRid=alpine.3.6-x64 -PortableBuild=false -strip-symbols",
+            "PB_AdditionalBuildArguments":"-TargetArchitecture=x64 -PortableBuild=false -strip-symbols",
+            "PB_AdditionalMSBuildArguments":"/p:OutputRid=alpine.3.6-x64",
```

Comment:
This shouldn't be necessary once we move up to the latest .NET Core SDK. However, since we are using 2.0.0 now to build, this is necessary.